images/sdn/Dockerfile.rhel8

```diff
@@ -0,0 +1,25 @@
+# This image contains the RHEL8 base and ovs/ovn fastdatapath RHEL8 rpms needed to set up ovs/ovn
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN INSTALL_PKGS=" \
+      which tar wget hostname shadow-utils \
+      socat findutils lsof bind-utils gzip \
+      procps-ng rsync ethtool socat nmap-ncat \
+      libmnl libnetfilter_conntrack \
+      libnfnetlink iproute procps-ng openssl \
+      binutils xz dbus \
+      openvswitch2.11 container-selinux \
+      openvswitch2.11-ovn-common openvswitch2.11-ovn-central \
+      openvswitch2.11-ovn-host openvswitch2.11-ovn-vtep \
+      openvswitch2.11-devel containernetworking-plugins \
+      conntrack-tools bridge-utils nftables sysvinit-tools \
+      " && \
+    if [ ! -e /usr/bin/dnf ]; then ln -s /usr/bin/microdnf /usr/bin/dnf; fi && \
```

Comment:
None of our other images are based on ubi-minimal; ubi7/ubi8 don't require microdnf. I'd be happier if this image didn't deviate, unless there's a specific reason for going with minimal.